tests/derives.rs

```diff
@@ -25,6 +25,17 @@ pub enum Color {
     Blue = 2,
 }
 
+#[open_enum]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[repr(u32)]
+pub enum ColorWithFeatures {
+    Red = 1,
+    Blue = 2,
+    /// Test doc
+    #[cfg(feature = "orange")]
```

Comment:
D'you have any ideas for testing that this works correctly with the `cfg` enabled?

Reply:
Hmm... we could maybe have a test enabled feature and have an enumeration with cfg enabled on it? Perhaps I should have done that in the PR. 

FWIW in the igvm repo we're testing both, but I can see about submitting a follow up PR to do the same here.